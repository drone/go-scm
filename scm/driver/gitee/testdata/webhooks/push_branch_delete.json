{
  "after": "0000000000000000000000000000000000000000",
  "before": "dc8fa2ebe050d63f639c5b834311e96bc2303523",
  "commits": [
  ],
  "commits_more_than_ten": null,
  "compare": "https://gitee.com/kit101/drone-yml-test/compare/dc8fa2ebe050d63f639c5b834311e96bc2303523...0000000000000000000000000000000000000000",
  "created": false,
  "deleted": true,
  "enterprise": null,
  "head_commit": null,
  "hook_id": 788005,
  "hook_name": "push_hooks",
  "hook_url": "https://gitee.com/kit101/drone-yml-test/hooks/788005/edit",
  "password": "",
  "project": {
    "clone_url": "https://gitee.com/kit101/drone-yml-test.git",
    "created_at": "2021-03-24T11:24:34+08:00",
    "default_branch": "master",
    "description": "",
    "fork": false,
    "forks_count": 0,
    "full_name": "kit101/drone-yml-test",
    "git_http_url": "https://gitee.com/kit101/drone-yml-test.git",
    "git_ssh_url": "git@gitee.com:kit101/drone-yml-test.git",
    "git_svn_url": "svn://gitee.com/kit101/drone-yml-test",
    "git_url": "git://gitee.com/kit101/drone-yml-test.git",
    "has_issues": true,
    "has_pages": false,
    "has_wiki": true,
    "homepage": "https://gitee.com/kit101/drone-yml-test",
    "html_url": "https://gitee.com/kit101/drone-yml-test",
    "id": 14836026,
    "language": null,
    "license": null,
    "name": "drone-yml-test",
    "name_with_namespace": "kit101/drone-yml-test",
    "namespace": "kit101",
    "open_issues_count": 2,
    "owner": {
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "email": "qkssk1711@163.com",
      "html_url": "https://gitee.com/kit101",
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "remark": null,
      "site_admin": false,
      "type": "User",
      "url": "https://gitee.com/kit101",
      "user_name": "kit101",
      "username": "kit101"
    },
    "path": "drone-yml-test",
    "path_with_namespace": "kit101/drone-yml-test",
    "private": false,
    "pushed_at": "2021-10-08T17:32:30+08:00",
    "ssh_url": "git@gitee.com:kit101/drone-yml-test.git",
    "stargazers_count": 0,
    "svn_url": "svn://gitee.com/kit101/drone-yml-test",
    "updated_at": "2021-10-08T17:32:30+08:00",
    "url": "https://gitee.com/kit101/drone-yml-test",
    "watchers_count": 1
  },
  "push_data": null,
  "pusher": {
    "email": "qkssk1711@163.com",
    "id": 1535738,
    "name": "kit101",
    "url": "https://gitee.com/kit101",
    "user_name": "kit101",
    "username": "kit101"
  },
  "ref": "refs/heads/feat-3",
  "repository": {
    "clone_url": "https://gitee.com/kit101/drone-yml-test.git",
    "created_at": "2021-03-24T11:24:34+08:00",
    "default_branch": "master",
    "description": "",
    "fork": false,
    "forks_count": 0,
    "full_name": "kit101/drone-yml-test",
    "git_http_url": "https://gitee.com/kit101/drone-yml-test.git",
    "git_ssh_url": "git@gitee.com:kit101/drone-yml-test.git",
    "git_svn_url": "svn://gitee.com/kit101/drone-yml-test",
    "git_url": "git://gitee.com/kit101/drone-yml-test.git",
    "has_issues": true,
    "has_pages": false,
    "has_wiki": true,
    "homepage": "https://gitee.com/kit101/drone-yml-test",
    "html_url": "https://gitee.com/kit101/drone-yml-test",
    "id": 14836026,
    "language": null,
    "license": null,
    "name": "drone-yml-test",
    "name_with_namespace": "kit101/drone-yml-test",
    "namespace": "kit101",
    "open_issues_count": 2,
    "owner": {
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "email": "qkssk1711@163.com",
      "html_url": "https://gitee.com/kit101",
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "remark": null,
      "site_admin": false,
      "type": "User",
      "url": "https://gitee.com/kit101",
      "user_name": "kit101",
      "username": "kit101"
    },
    "path": "drone-yml-test",
    "path_with_namespace": "kit101/drone-yml-test",
    "private": false,
    "pushed_at": "2021-10-08T17:32:30+08:00",
    "ssh_url": "git@gitee.com:kit101/drone-yml-test.git",
    "stargazers_count": 0,
    "svn_url": "svn://gitee.com/kit101/drone-yml-test",
    "updated_at": "2021-10-08T17:32:30+08:00",
    "url": "https://gitee.com/kit101/drone-yml-test",
    "watchers_count": 1
  },
  "sender": {
    "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
    "email": "qkssk1711@163.com",
    "html_url": "https://gitee.com/kit101",
    "id": 1535738,
    "login": "kit101",
    "name": "kit101",
    "remark": null,
    "site_admin": false,
    "type": "User",
    "url": "https://gitee.com/kit101",
    "user_name": "kit101",
    "username": "kit101"
  },
  "sign": "l9f7bD44j3IX+WILn5LlyNy8E37v4xP1xB6eaiMGa5A=",
  "timestamp": "1633685551787",
  "total_commits_count": null,
  "user": {
    "email": "qkssk1711@163.com",
    "id": 1535738,
    "name": "kit101",
    "url": "https://gitee.com/kit101",
    "user_name": "kit101",
    "username": "kit101"
  },
  "user_id": 1535738,
  "user_name": "kit101"
}