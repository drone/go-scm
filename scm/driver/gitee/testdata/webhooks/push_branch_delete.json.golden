{
  "Ref": "refs/heads/feat-3",
  "Before": "dc8fa2ebe050d63f639c5b834311e96bc2303523",
  "After": "0000000000000000000000000000000000000000",
  "Repo": {
    "ID": "14836026",
    "Namespace": "kit101",
    "Name": "drone-yml-test",
    "Perm": null,
    "Branch": "master",
    "Private": false,
    "Clone": "https://gitee.com/kit101/drone-yml-test.git",
    "CloneSSH": "git@gitee.com:kit101/drone-yml-test.git",
    "Link": "https://gitee.com/kit101/drone-yml-test",
    "Created": "2021-03-24T11:24:34+08:00",
    "Updated": "2021-10-08T17:32:30+08:00"
  },
  "Commit": {
    "Sha": "0000000000000000000000000000000000000000",
    "Link": "https://gitee.com/kit101/drone-yml-test/compare/dc8fa2ebe050d63f639c5b834311e96bc2303523...0000000000000000000000000000000000000000"
  },
  "Sender": {
    "Avatar": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
    "Email": "qkssk1711@163.com",
    "Login": "kit101",
    "Name": "kit101"
  }
}